enterprise/api_client/enterprise_catalog.py

```diff
@@ -128,6 +128,13 @@ def get_content_metadata(self, enterprise_customer, enterprise_catalogs=None):
             endpoint = getattr(self.client, self.GET_CONTENT_METADATA_ENDPOINT.format(catalog_uuid))
             try:
                 response = endpoint.get()
+                results = utils.traverse_pagination(response, endpoint)
```

Comment:
You likely already know this, but just a reminder that this won't work until the pagination for `get_content_metadata` in enterprise catalog is actually implemented